docs/voltohandson/header.md

```diff
@@ -94,6 +94,11 @@ From that point on, the hot reloading should kick in and reload the page automat
 We will customize the existing Volto header, since the one we want does not differ much from the original.
 We will do so by copying the original Volto `Header` component from the `omelette` folder `omelette/src/components/theme/Header/Header.jsx` folder into `src/customizations/components/theme/Header/Header.jsx`.
 
+```{note}
+If you have not worked with React that much so far you will notice that the Navigation component in Volto is not a javascipt function. This is because in React components can also be created from a js [class](https://reactjs.org/docs/react-component.html). Actually this was the preffered way to create components in earlier versions of React. Volto is currently undergoing the progress to switch to function components where possible.
```

Comment:
```suggestion
If you have not worked with React much, you will notice that the Navigation component in Volto is not a JavaScript function.
This is because in React components can also be created from a JavaScript [class](https://reactjs.org/docs/react-component.html).
Actually this was the preferred way to create components in earlier versions of React.
Volto is currently undergoing the procress to switch to function components where possible.
```